circuits/cpp/src/aztec3/circuits/rollup/base/.test.cpp

```diff
@@ -148,7 +151,7 @@ TEST_F(base_rollup_tests, native_no_new_contract_leafs)
     ASSERT_EQ(outputs.start_contract_tree_snapshot, expectedStartContractTreeSnapshot);
     ASSERT_EQ(outputs.end_contract_tree_snapshot, expectedEndContractTreeSnapshot);
     ASSERT_EQ(outputs.start_contract_tree_snapshot, emptyInputs.start_contract_tree_snapshot);
-    ASSERT_FALSE(builder.failed());
+    ASSERT_FALSE(builder.failed()) << builder.failure_msgs;
```

Comment:
@jeanmon is this your `<< builder.failure_msgs`? There is a few of them that have sneaked into the codebase, is this on purpose?

Reply:
Yes this is on purpose. This the way gtest works (or at least my understanding of it) to log additional information when a test is failing.

Reply:
@LHerskind Do you see any issue with this?

Reply:
Were just that it is extra print things that we don't expect to normally need, but if only when it fails, think is probably fine 👍. Thanks for clarifying

Reply:
Exactly.